gui/camera/scene.py

```diff
@@ -0,0 +1,59 @@
+from qtpy.QtGui import QCursor
+from qtpy.QtWidgets import QGraphicsView
+from qtpy.QtCore import Signal, Qt
+import daq_utils
+import typing
+
+if typing.TYPE_CHECKING:
+    from gui.control_main import ControlMain
+
+
+class CustomView(QGraphicsView):
```

Comment:
Created CustomView mainly to handle events that happen when the sample cam is in focus. It handles the following events:
- Using the mouse wheel (connects to `handle_wheel` method in `gui.camera.zoom_widget.ZoomSlider`
- Detects whether Ctrl key is pressed and the mouse is clicked on the sample cam
- Triggers center_on_click when all conditions above are met